datastore/google/cloud/datastore/helpers.py

```diff
@@ -19,16 +19,17 @@
 
 import datetime
 import itertools
-
-from google.protobuf import struct_pb2
-from google.type import latlng_pb2
 import six
 
 from google.cloud._helpers import _datetime_to_pb_timestamp
 from google.cloud._helpers import _pb_timestamp_to_datetime
-from google.cloud.proto.datastore.v1 import entity_pb2 as _entity_pb2
 from google.cloud.datastore.entity import Entity
 from google.cloud.datastore.key import Key
+from google.cloud.proto.datastore.v1 import entity_pb2 as _entity_pb2
+from google.cloud.proto.datastore.v1 import datastore_pb2 as _datastore_pb2
```

Comment:
You don't need to do this import alias-ing at this point. The reason the other ones are aliased are historical: there was a code change from `v1beta3` to `v1` and we didn't want to disturb the existing code.

Reply:
So just 
google.cloud.proto.datastore.v1 import entity_pb2
google.cloud.proto.datastore.v1 import datastore_pb2?

Reply:
Yes